### DISCUSSION TITLE
Welcome to Toughest Discussions!

### DISCUSSION THREAD

**Discussion by MDSADABWASIM** [MDSADABWASIM/Toughest#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kunalmanocha98** [comment]
How to be a contributor of this project?

**Reply by MDSADABWASIM** [reply]
Hi @kunalmanocha98 , Simply fork the project and do the changes and submit a pull request.
Thanks.
